src/components/ebay-skeleton/examples/withContent.marko

```diff
@@ -0,0 +1,38 @@
+class {
+    onCreate() {
+        this.state = {
+            isLoading: true,
+            data: null,
+        };
+    }
+    async getData() {
+        return new Promise((resolve, reject) => {
+            setTimeout(() => resolve("Button"), 2000);
+        });
+    }
+    async onMount() {
+        try {
+            this.state.data = await this.getData();
+        } catch (err) {
+            this.state.error = err;
+        } finally {
+            this.state.isLoading = false;
+        }
+    }
+}
+
+<style>
+    .demo-container {
+        width: 88px;
+    }
+</style>
+<div class="demo-container">
```

Comment:
Id rename this more specific. I believe this might be used by storybook or in other places. 
Maybe `skeleton-example-container`

Reply:
Done, renamed consistent to other story class names.